todo_cli/cli/cli_controller.py

```diff
@@ -142,3 +152,28 @@ def delete_many(task_id: List[str] = typer.Argument(None)) -> None:
         raise typer.Exit(code=3)
 
     delete_task(task_id)
+
+
+@app.command()
+def export(file_format: Annotated[Formats,
+                             typer.Option(case_sensitive=False, prompt=True)],
+           file: Annotated[typer.FileTextWrite, typer.Option(prompt=True)]):
+    """
+    This function will export your data to either a JSON or CSV format
+
+    args:
+        format(str): This is the format in which you would like to export the file.
+        file(str): This is the file you would like to export your data to.
+    """
+    content = read_tasks_file()
+
+    if file_format == Formats.JSON:
+        json.dump(content, file, indent=4)
+    elif file_format == Formats.CSV:
+        writer = csv.writer(file)
+        writer.writerow(['UUID', 'name', 'date', 'priority'])
+        for task in content:
+            writer.writerow([task['UUID'], task['name'],
+                            task['date'], task['priority']])
```

Comment:
No changes required here, but in future maybe we should have different functions for JSON exporting and CSV exporting. 

We have quite nested logic here and should also try and create functions that follow the single responsibility principle.